library/src/main/java/me/grantland/widget/AutofitHelper.java

```diff
@@ -232,8 +262,9 @@ else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
     private float mMaxTextSize;
     private float mPrecision;
 
-    private boolean mEnabled;
+    private boolean mIsAutofitWidthEnabled;
     private boolean mIsAutofitting;
+    private static boolean mIsAutofitHeightEnabled;
```

Comment:
Why `AutofitHelper::mIsAutofitHeightEnabled` is a static variable?

If I use two different `AutofitTextView` in my app, one with `AutofitHeight` enabled, and a second with `AutofitHeight` disabled, then the first will get `AutofitHeight` disabled too. Is this the expected behaviour?